packages/larva/scripts/larva/larva.config.js

```diff
@@ -0,0 +1,42 @@
+const path = require( 'path' );
+
+module.exports = {
+	webpack: {
+		aliases: {
+			'@js': path.resolve( './src/js' )
+		},
+		entries: {
+			common: './entries/common.entry.js'
+		}
+	},
+
+	backstop: {
+		// testBaseUrl: 'https://notlaura.com',
+		testBaseUrl: 'http://localhost:3000/project',
+		testScenario: {
+			'delay': 1000,
+			'misMatchThreshold': 0.5,
+		},
+		// testPaths: [ '/about/' ],
+		larvaModules: [ 'footer', 'breadcrumbs' ],
+		backstopConfig: {
+			'engineOptions': {
+				'args': [ '--no-sandbox', '--proxy-server=127.0.0.1:3000', '--proxy-bypass-list=<-loopback>' ],
+			}
+		}
+	},
+
+	patterns: {
+		larvaPatternsDir: path.resolve( './node_modules/@penskemediacorp/larva-patterns' ),
+		projectPatternsDir: path.resolve( './src/patterns' ),
+		ignoredModules: [
+			'footer-menus',
+			'footer-social',
+			'footer-tip'
+		]
+	},
+
+	parser: {
```

Comment:
we want this here even though has no values?

Reply:
I need to add a default value for the parser I think - need to test it. Also will do in a clean up PR.